### DISCUSSION TITLE
[Support]:  Is it possible to create a new audio detection model?

### DISCUSSION THREAD

**Discussion by sgrossiP** [blakeblackshear/frigate#14594] [General Support]
### Describe the problem you are having

I would like to have an audio model that detects two, three or 4 claps. the current ones for hands, clapping or applause do not meet my requirements

### Version

0.13.1

### What browser(s) are you using?

130.0.6723.58

### Frigate config file

```yaml
no relevant to this question
```

### Relevant Frigate log output

```shell
no relevant to this question
```

### Relevant go2rtc log output

```shell
no relevant to this question
```

### FFprobe output from your camera

```shell
no relevant to this question
```

### Install method

Docker Compose

### docker-compose file or Docker CLI command

```yaml
no relevant to this question
```

### Object Detector

CPU (no coral)

### Network connection

Wired

### Camera make and model

no relevant to this question

**Comment by NickM-27** [comment]
that seems very specific and not really how the model works, it is also outside the scope of what frigate uses audio detection for

**Reply by sgrossiP** [reply]
you are right :(
